minard/detectorviz.py

```diff
@@ -0,0 +1,143 @@
+""" module for code relating to detector visualization image generation """
+from PIL import Image
```

Comment:
Hmm yeah I've had no luck installing it on minard, so if you have a work around that would be preferred. Otherwise, we can ask the network team to try and get it working.

Reply:
right, I have also tried and couldn't get it to install but the issue wasn't with the module but with ssl certificate (ie the server couldn't really reach web). I'll ask the network team.